src/components/EnterpriseApp/EnterpriseAppRoutes.jsx

```diff
@@ -105,6 +106,11 @@ const EnterpriseAppRoutes = ({
         />
       )}
 
+        <Route
```

Comment:
[suggestion] Given this `Route` and the above `Route` both relate to the same Learner Credit Management (LCM) feature, I'm wondering if this component should be defining a single parent route for the overall LCM feature, and then the component passed to the parent route has its owned rendered sub-routes for the LCM-specific routes. That way, `EnterpriseAppRoutes`, doesn't really need any knowledge of the detail route.

Related, as is, this LCM detail page route would be available even when the LCM overview page route (defined above) is not since it isn't relying on the `canManageLearnerCredit` variable. If the overview page route isn't available; neither should the detail page route.